src/index.html

```diff
@@ -1,22 +1,34 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    />
-    <meta
-      http-equiv="X-UA-Compatible"
-      content="ie=edge"
-    />
-    <title>Moyo header</title>
-    <link
-      rel="stylesheet"
-      href="./style.css"
-    />
-  </head>
-  <body>
-    <h1>Moyo header</h1>
-  </body>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Moyo Header</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+</head>
+<body>
+
+<header>
+    <div class="logo">
+        <a href="#">
+            <img src="images/logo.png" alt="Logo">
+        </a>
+    </div>
+    <nav>
+        <ul>
+            <li><a href="#">APPLE</a></li>
+            <li><a href="#">SAMSUNG</a></li>
+            <li><a href="#">LAPTOPS & COMPUTERS</a></li>
```

Comment:
This item should have a `data-qa="hover"` attribute [Reference](https://github.com/mate-academy/layout_moyo-header#:~:text=%E2%9D%97%EF%B8%8F%20add%20data%2Dqa%3D%22hover%22%20attribute%20to%20the%204th%20link%20for%20testing%20(Laptops%20%26%20computers))